src/main/java/edu/hm/hafner/analysis/parser/PyLintParser.java

```diff
@@ -41,6 +41,14 @@ protected boolean isLineInteresting(final String line) {
         builder.setSeverity(mapPriority(category));
         builder.setCategory(StringUtils.firstNonBlank(matcher.group("symbol"), category, UNKNOWN_CAT));
 
+        final String moduleName = matcher.group("module");
+        if (moduleName != null) {
+            if (moduleName.contains(".")) {
+                builder.setPackageName(moduleName.substring(0, moduleName.lastIndexOf(".")));
+            }
+            builder.setModuleName(moduleName);
+        }
+
```

Comment:
Here you need:
```
else {
   builder.setPackageName("-").setModuleName("-");
}
```

Reply:
The builder is not cleared between warnings.

Reply:
Aaa, that confirms what I was seeing, effectively. I had the impression something wasn't cleared, but didn't knew what it was. I'll adjust this, thanks :)

Reply:
Fixed.